cli/ballerina-cli/src/main/resources/cli-help/ballerina-bindgen.help

```diff
@@ -13,32 +13,34 @@ DESCRIPTION
        Generate Ballerina bridge code for Java APIs.
 
        Ballerina bindings could be generated for Java classes residing inside Java libraries
-       or for standard Java classes. The Java classes will be mapped to the Ballerina
-       classes providing a seamless Java interoperability developer experience to Ballerina users.
+       or standard Java classes. The Java classes will be mapped onto Ballerina
+       objects making the developer experience of Ballerina Java interoperability seamless.
 
-       In addition to the user-specified Java classes, partial implementations of directly
-       -dependent Java classes will also be generated by the tool. By default, the bindings for
-       each Java package will be mapped to a separate Ballerina module.
+       The directly-dependent Java classes and other required resources will be automatically
+       generated apart from the specified Java classes.
```

Comment:
Why drop the remark about bindings of a Java package being mapped to a Ballerina module? Even the other one, conveys a completely different meaning to what the original text conveyed. We are not generating Java classes. We are generating Ballerina code for the Java classes.

Reply:
@IrushiL

Reply:
Yeah seems like the older version is restored after resolving the conflicting PR.

Shall we change that as,

**In addition to the user-specified Java classes, partially implemented bindings for dependent Java classes will also be generated by the tool. By default, the bindings for each Java package will be mapped to a separate Ballerina module.**

Reply:
Updated.